packages/ifpack2/src/Ifpack2_SparseContainer_def.hpp

```diff
@@ -141,13 +204,39 @@ void SparseContainer<MatrixType,InverseType>::initialize ()
   // local matrices to use for solves.
   diagBlocks_.reserve(this->numBlocks_);
   Inverses_.reserve(this->numBlocks_);
+
+  const size_t maxNumEntriesInRow = this->inputMatrix_->getNodeMaxNumRowEntries();
+  Teuchos::Array<scalar_type> Values;
+  Teuchos::Array<local_ordinal_type> Indices;
+  Teuchos::Array<InverseScalar> Values_insert;
+  Teuchos::Array<InverseGlobalOrdinal> Indices_insert;
+  Values.resize (maxNumEntriesInRow);
```

Comment:
The constructor of `Teuchos::Array`, just like `std::vector`, takes an integer size argument.

Reply:
Done.